src/components/ManifestEntitlementsList/ManifestEntitlementsListContainer.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import ManifestEntitlementsList from './ManifestEntitlementsList';
+import useManifestEntitlements from '../../hooks/useManifestEntitlements';
+
+interface ManifestEntitlementsListContainerProps {
+  uuid: string;
+}
+const ManifestEntitlementsListContainer: FC<ManifestEntitlementsListContainerProps> = ({
```

Comment:
The reason I've been splitting these up is to make them more testable.  So far, I've been separating anything that has a react-query data-fetching call into one that just handles the data, and then a "dumb" component that is just rendering that data passed as props.  

We can technically have it be one component, but it will require somewhat bloated mocks for tests for each scenario.

Reply:
It seems like any mocking could be done in just one beforeEach block.  And the logic of the data fetching is already extracted out into the hook file you created.  It just feels like one more layer than is needed.

I guess the component would technically be harder to reuse without the ability to pass it data directly though.

I don't know.